### DISCUSSION TITLE
I'm tired of this app router

### DISCUSSION THREAD

**Discussion by abdoachhoubi** [vercel/next.js#54625] [App Router]
The app router in the current version of NextJS appears to be excessively complex and challenging to comprehend and troubleshoot. This complexity is resulting in a significantly negative development experience. I am dissatisfied with certain updates introduced in Next.js 13. Personally, I find the previous version of Next.js to be more straightforward to configure and less prone to errors.

**Comment by wilsonneto-swe** [comment]
Yes, still there are some problems... But, it's part of the progress, continuous improvements in the first versions of page routing it wasn't that good. 
But, even in the new versions, you have the option to use the page style routing. Also, even the page style is a bit confusing and challenging for the first-timers.

Are you able to select between pages and apps when you run Create Next App?

Also, please list the main pain points, so the Vercel/Next.js team will be able to focus the effort and help the community better.
